libs/cmdctx/account_client.go

```diff
@@ -7,6 +7,9 @@ import (
 )
 
 func SetAccountClient(ctx context.Context, a *databricks.AccountClient) context.Context {
+	if v := ctx.Value(accountClientKey); v != nil {
+		panic("cmdctx.SetAccountClient called twice on the same context.")
+	}
```

Comment:
Is there corresponding test coverage? I recall seeing panic tests for the other setters.

Reply:
Thanks, added test coverage.